### PR TITLE
[RHACS] [rhacs-docs-4.4] ROX-23460: Adding 4.4.1 RN

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.4

Issue:
https://issues.redhat.com/browse/ROX-23460

Link to docs preview:
https://74570--ocpdocs-pr.netlify.app/openshift-acs/latest/release_notes/44-release-notes#resolved-in-version-441_release-notes-44

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-4.4`

### CODE REVIEW

File: release_notes/44-release-notes.adoc [hunk 1]
```diff
@@ -561,6 +562,28 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
...
+
+
+[id="known-issues-440_{context}"]
+== Known issues
+
+*Updated on*: 15 April 2024
+
+* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where Collector uses the EBPF collection method when CORE_BPF is selected from {product-title-short} 4.4.1 and later.
++
+Workaround: If you use OpenShift Container Platform 4.12 Power offline, you must use a support package.
```
Comment (ovalenti) [inline release_notes/44-release-notes.adoc:585]:
```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where CORE_BPF collection method is not functioning properly. From {product-title-short} 4.4.1 and later, in this particular case, Collector uses automatically the EBPF collection method when CORE_BPF is selected.

Impact: If you use OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.
```

**Review by ovalenti — CHANGES_REQUESTED** [review]
I expanded slightly to give more context, and replaced the term `workaround` since this known issue is about the limitations of a workaround; which seemed a bit misleading.

**Comment by agantony** [comment]
> I expanded slightly to give more context, and replaced the term `workaround` since this known issue is about the limitations of a workaround; which seemed a bit misleading.

@ovalenti  I have incorporated your suggestions. If the changes look good to you, please approve the PR. Thanks!

**Review by ovalenti — APPROVED** [review]
The "known issues" section looks good to me, thanks 👍 

I read through the other parts and it seems alright, but I cannot validate the content.

File: release_notes/44-release-notes.adoc [hunk 2]
```diff
@@ -561,6 +562,27 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
...
 
+
+[id="resolved-in-version-441_{context}"]
+=== Resolved in version 4.4.1
+
+*Release date*: 15 April 2024
+
+* With this update, rendering the Helm charts `stackrox-central-services` and `stackrox-secured-cluster-services` requires a Helm version 3.9.0 or later, as support for older Helm versions has been discontinued.
+
+* Fixed an issue where {product-title-short} would not qualify the registry and path of images from the container runtime if the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` environment variable was set to `true` on both Central and Sensor. This update now enables support for CRI-O's unqualified search registries and short name aliases.
```
Comment (dhaus67) [inline release_notes/44-release-notes.adoc:573]: I thought we do not document this - @dcaravel to verify whether this should be here or not.

**Review by dhaus67 — APPROVED** [review]
The rest of the release note section looks good to me; I have one comment regarding the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` - pinged @dcaravel to confirm, please do not merge without his approval.

File: release_notes/44-release-notes.adoc (hunk 2 above)
Comment (dcaravel) [reply release_notes/44-release-notes.adoc:573]: That's correct, please remove the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` section for now. Thanks @agantony

Comment (agantony) [reply release_notes/44-release-notes.adoc]: ack

**Review by dcaravel — APPROVED** [review]

File: release_notes/44-release-notes.adoc [hunk 3]
```diff
@@ -561,6 +571,23 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
...
+*Release date*: 15 April 2024
+
+* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.
+
+[id="known-issues-440_{context}"]
+== Known issues
+
+*Updated on*: 15 April 2024
+
+* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected.
```
Comment (kcarmichael08) [inline release_notes/44-release-notes.adoc:587]:
```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the IBM Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
```

File: release_notes/44-release-notes.adoc [hunk 4]
```diff
@@ -561,6 +571,23 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
...
+* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.
+
+[id="known-issues-440_{context}"]
+== Known issues
+
+*Updated on*: 15 April 2024
+
+* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
+
+If you are using OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.
```
Comment (kcarmichael08) [inline release_notes/44-release-notes.adoc:589]:
```suggestion
If you are using OpenShift Container Platform 4.12 on the IBM Power architecture and in offline mode, you must use a support package.
```

**Review by kcarmichael08 — COMMENTED** [review]
Couple of minor things, otherwise lgtm!

**Comment by kcarmichael08** [comment]
This is ready to merge, but I think we need to wait for Monday 4/15 until the release goes out.

**Pull request merged**